tools/esdlcmd/esdl-publish.cpp

```diff
@@ -94,8 +93,6 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
             return true;
         if (iter.matchFlag(optWSProcPort, ESDL_OPTION_SERVICE_PORT) || iter.matchFlag(optWSProcPort, ESDL_OPT_SERVICE_PORT))
             return true;
-        if (iter.matchFlag(optVersionStr, ESDLOPT_VERSION))
```

Comment:
Also, while we're at it...
It seems the usage at the top level might be off:
esdlcmd_shell.cpp:   
esdl [--version] <command> [<args>]\n\n"
should change to 
  esdl <command> [<args>]\n\n"
and --version should be listed as a command

@JamesDeFabia this change might require a doc change (I think an issue is already open, but wanted to point this out)